docs/source/contributing.rst

```diff
@@ -90,12 +92,17 @@ Submit a request::
                                      --amount 1 \
                                      --target-address 0x1CEE82EEd89Bd5Be5bf2507a92a755dcF1D8e8dc \
                                      --target-chain-id 1337 \
-                                     --target-token-address $TOKEN_ADDR
+                                     --target-token-address $TOKEN_ADDR \
+                                     --validity-period 2700
```

Comment:
Why do we need `--validity-period` option here?

Reply:
Then I would rather fix the script and change the default value to be valid, according to the current min/max validity in the contract. This way, no option would be needed. I suggest to just set it to the current minimum validity period, which is 30 min.